lib/core/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -15,7 +15,7 @@
 
                     <adf-select-filter-input *ngIf="showInputFilter" (change)="onFilterInputChange($event)"></adf-select-filter-input>
 
-                    <mat-option *ngIf="showNoneOption()">{{ 'CORE.CARDVIEW.NONE' | translate }}</mat-option>
+                    <mat-option *ngIf="showNoneOption() && property.key !== 'nodeType'">{{ 'CORE.CARDVIEW.NONE' | translate }}</mat-option>
```

Comment:
The problem here is `Cardviewselectmodel` doesn't handle for `dispalynone` option. So the fix will be updating the [model]( https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/core/card-view/models/card-view-selectitem.model.ts#L25) and use this while constructing the cardview[ here]( https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/content-services/src/lib/content-metadata/services/content-type-property.service.ts#L83)

Reply:
@dhrn , I don't understand why displayNoneOption should be part of the model. Can you explain a bit? What use case do you have in mind?